examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>Maybe consider having some sort of timeout built into your loop? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>